pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/realtime/PinotLLCRealtimeSegmentManager.java

```diff
@@ -504,6 +504,8 @@ private LLCRealtimeSegmentZKMetadata updateCommittingSegmentZKMetadata(String re
     committingSegmentZKMetadata.setDownloadUrl(isPeerURL(committingSegmentDescriptor.getSegmentLocation())
         ? CommonConstants.Segment.METADATA_URI_FOR_PEER_DOWNLOAD : committingSegmentDescriptor.getSegmentLocation());
     committingSegmentZKMetadata.setCrc(Long.valueOf(segmentMetadata.getCrc()));
+    Preconditions.checkNotNull(segmentMetadata.getTimeInterval(),
```

Comment:
Consumption will stop if this condition hits. Are we sure that is what we want?

Reply:
@mcvsubbu This condition should not happen after the fix from #5966. When this happens, the realtime segment commit would anyway fail and the consumption would be stopped due to the null pointer exception. (`egmentMetadata.getTimeInterval().getStartMillis()` gets called at the following line). I'm just trying to add more information to the logs here since the log message was not informative when we hit that.

We recently saw the null pointer error issue (and the consumption actually stopped) when the time column is misconfigured. 
```
2020/09/17 20:36:52.666 ERROR [SegmentCompletionFSM_XXXXX__3__0__20200915T2142Z] [grizzly-http-server-3] [pinot-controller] [] Caught exception while committing segment metadata for segment: XXXXX__3__0__20200915T2142Z
java.lang.NullPointerException: null
```